### PR TITLE
Mid Term Blog: Tutorials

### PR DESCRIPTION
**Pull request by shwetalsoni**
Screenshot from 2021-08-03 16-10-43
Screenshot from 2021-08-03 16-11-11
Screenshot from 2021-08-03 16-12-10
Screenshot from 2021-08-03 16-12-19
Screenshot from 2021-08-03 16-12-38
Screenshot from 2021-08-03 16-12-55
Screenshot from 2021-08-03 16-13-04
Screenshot from 2021-08-03 16-13-14
Screenshot from 2021-08-03 16-13-28
Screenshot from 2021-08-03 16-13-37
Screenshot from 2021-08-03 16-13-45
Screenshot from 2021-08-03 16-13-56
Screenshot from 2021-08-03 16-14-10

### CODE REVIEW

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]
I feel it's not a good idea to put your email in the main heading along with your name, but anyways as this not a documentation for Wechaty, you can keep that any way you wish to keep it.

**Comment by shwetalsoni** [comment]
> I feel it's not a good idea to put your email in the main heading along with your name, but anyways as this not a documentation for Wechaty, you can keep that any way you wish to keep it.

Ok, I'll keep that in mind from next time :)

**Review by proudofsimin — APPROVED** [review]
Thank you.

**Review by sbis04 — APPROVED** [review]

**Comment by shwetalsoni** [comment]
@Rohitesh-Kumar-Jain and @proudofsimin please add this PR to the merging list

Thank you :)

File: jekyll/_posts/2021-08-03-gsod-2021-mid-term-blog-tutorials.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
+---
+title: "Google Season of Docs 2021 Mid Term Blog: Tutorials"
+author: shwetalsoni
+categories: tutorial
```
Comment (lijiarui) [inline jekyll/_posts/2021-08-03-gsod-2021-mid-term-blog-tutorials.md]: Please change the categories to `gsod` here.

**Review by lijiarui — CHANGES_REQUESTED** [review]
Pls change as I request, thanks!

File: jekyll/_posts/2021-08-03-gsod-2021-mid-term-blog-tutorials.md (hunk 1 above)
Comment (shwetalsoni) [reply jekyll/_posts/2021-08-03-gsod-2021-mid-term-blog-tutorials.md]: @lijiarui I have applied the requested changes.

**Review by lijiarui — APPROVED** [review]
LGTM

**Pull request merged**